packages/alea-frontend/pages/api/reset-password.ts

```diff
@@ -9,14 +9,14 @@ export default async function handler(
 ) {
   const { email, resetPasswordToken, newPassword } = req.body;
   const userPasswordResetInfo = (await executeAndEndSet500OnError(
-    `SELECT passwordResetToken FROM userInfo WHERE userId = ?`,
+    `SELECT passwordResetToken , passwordResetTokenGeneratedAt FROM userInfo WHERE userId = ?`,
```

Comment:
the sql file where this field is added is missing in this PR. Try to make this field's type `Timestamp`. If that's not possible, the name needs to be more descriptive `passwordResetRequestTimestampMs`